llvm/lib/Transforms/Utils/CodeExtractor.cpp

```diff
@@ -627,6 +627,31 @@ bool CodeExtractor::isEligible() const {
         return false;
     }
   }
+  // stacksave as input implies stackrestore in the outlined function
+  // This can confuse prologue epilogue insertion phase
+  // stacksave's uses must not cross outlined function
```

Comment:
```suggestion
  // stacksave as input implies stackrestore in the outlined function.
  // This can confuse prologue epilogue insertion phase.
  // stacksave's uses must not cross outlined function.
```

Might also want to reflow comment